src/main/webapp/WEB-INF/bb-manifest.xml

```diff
@@ -47,8 +47,9 @@
             <permission type="java.util.logging.LoggingPermission" name="control" actions="" />
             <permission type="java.lang.RuntimePermission" name="db.connection.bb_bb60" actions="persist" />
             <permission type="java.lang.reflect.ReflectPermission" name="suppressAccessChecks" />
-            <permission type="java.util.PropertyPermission" name="*" actions="read,write" />
-            <permission type="java.io.FilePermission" name="&lt;&lt;ALL FILES&gt;&gt;" actions="read,write,delete" />
+            <permission type="java.util.PropertyPermission" name="*" actions="read" />
+            <!-- blackboard/ is the home dir.: https://help.blackboard.com/Learn/Administrator/Hosting/Architecture/About_the_File_System -->
+            <permission type="java.io.FilePermission" name="blackboard/-" actions="read,write,delete" />
```

Comment:
Volgens mij klopt dit niet. Deze regels kloppen volgens mij beter?

https://github.com/mark-b-kauffman/bbdn-bblogbackb2/blob/master/src/main/webapp/WEB-INF/bb-manifest.xml#L129-L130

Reply:
> https://github.com/mark-b-kauffman/bbdn-bblogbackb2/blob/master/src/main/webapp/WEB-INF/bb-manifest.xml#L129-L130

Die eerste regel met de `${java.home}` property zorgt ervoor dat het niet build. Heb jij enig idee waardoor het kan komen? Ik kan moeilijk wat uit de Gradle logs halen naast een totaal generieke:

```
Execution failed for task ':war'.
> Could not add file 'C:\projects\Java\blackboard-mytimetable\src\main\webapp\WEB-INF\bb-manifest.xml' to ZIP 'C:\
projects\Java\blackboard-mytimetable\build\libs\blackboard-mytimetable-2.2.0.war'.
```

Maar als ik hem eruit laat, dan build het weer.

Reply:
Dat heeft hiermee te maken denk ik, maar meer hints heb ik ook niet voor je: https://github.com/eveoh/blackboard-mytimetable/blob/master/build.gradle#L34

Die wil dingen met een `$` expanden, en mogelijk moet daar dus iets escaped worden of anders gedaan worden.

Reply:
Thanks, dat was genoeg.